ONSdigital/rasrm-acceptance-tests
# Tests surrounding account unlocking verification and password reset

# Motivation and Context
User journeys surrounding resetting password/verification and unlocking account weren't tested. 
So we added in a bunch of features that cover these journeys.

# What has changed
New features added in 

# How to test?
Check all scenarios covered from this [Card](https://trello.com/c/gmplQz3g/413-acceptance-tests-surrounding-sign-in-verification-account-locking-unlocking-l)

And make sure all AT's pass
